### PR TITLE
API Functionality

### PR DESCRIPTION
**Pull request by AKeeganDev**
This pull request features the following updates:

- The page loads with data from the new leaderboard for "Aaron Game"
- When a user enters a name and a score to the appropriate input fields then the data is sent to the API for storage
- When the user clicks on the Refresh button the leaderboard refreshes and places the new player's score in the appropriate position on the leaderboard.
- The leaderboard is set to have the highest score at the top followed by the next closest scores

### CODE REVIEW

File: src/modules/pageManager.js [hunk 1]
```diff
@@ -12,5 +13,46 @@ export default class PageManager {
     });
   }
 
+  pullScoresFromAPI = () => {
+    const allScores = Scores.getAPI();
+    allScores.then((scores) => {
+      Scores.scores = [];
+      this.scoreContainer.innerHTML = '';
+      scores.forEach((score) => {
+        console.log(`${score.user}: ${score.score}`);
```
Comment (Meltrust) [inline src/modules/pageManager.js]: - [ ] Please, kindly delete this debugging `console.log` statement from the repo, according to the Javascript required best practices.

File: src/modules/scores.js [hunk 2]
```diff
@@ -1,7 +1,30 @@
...
+      },
+      body: JSON.stringify({
+        user: `${userName}`,
+        score: `${score}`,
+      }),
+    });
 
-  static getScores = () => Scores.scores;
+    const data = await response.json();
+    console.log(data);
```
Comment (Meltrust) [inline src/modules/scores.js]: - [ ] Please, kindly delete this debugging `console.log` statement from the repo, according to the Javascript required best practices.

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @AKeeganDev

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- The app works well!✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/scores.js (hunk 2 above)
Comment (AKeeganDev) [reply src/modules/scores.js]: @Meltrust 

Oops. Got it, thanks

**Review by Kingstalux — APPROVED** [review]
Hi @AKeeganDev,
# Status: Approved ✔️ ✔️ ✔️ 
 
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966

Good job implementing the required changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [ ] (OPTIONAL) You have not set up the Eslint linter which is required as you are working with JavaScript. Kindly set up the [Linters](https://github.com/microverseinc/linters-config/tree/master/html-css-js) and make sure the checks show up on GitHub and they are passing.
- [ ] (OPTIONAL) It would be great to provide the necessary commands (`npm install`,` npm start`) needed to set up and run your project locally in your README file in order to make it more professional 💯 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**